### PR TITLE
Hotfix/projects disappearing

### LINKED ISSUE #111: Projects get removed from the search if some order is chosen
Showcase

### PR DESCRIPTION
**Pull request by aabounegm**
Fixes #111

### CODE REVIEW

File: src/components/projects/project-card.svelte [hunk 1]
```diff
@@ -14,10 +14,12 @@
   export let start_date;
   export let end_date;
   export let moderated = false;
+
+  const image_placeholder = '/images/create-product/placeholder.svg';
 </script>
 
 <Card classname="with-image">
-  <img src="{API_HOST}/file/{image_id}" alt="" class="image" />
+  <img src={image_id == null ? image_placeholder :`${API_HOST}/file/${image_id}`} alt="" class="image" />
```
Comment (illright) [inline src/components/projects/project-card.svelte]: Spaces after the colon, gentleman

File: src/components/projects/project-card.svelte [hunk 2]
```diff
@@ -14,10 +14,12 @@
   export let start_date;
   export let end_date;
   export let moderated = false;
+
+  const image_placeholder = '/images/create-product/placeholder.svg';
```
Comment (illright) [inline src/components/projects/project-card.svelte]: Also I'd use lowerCamelCase here

Comment (aabounegm) [reply src/components/projects/project-card.svelte]: ugh, been working with Python a lot lately

File: src/components/projects/project-card.svelte (hunk 1 above)
Comment (aabounegm) [reply src/components/projects/project-card.svelte]: and that's what happens when you try to rush the PR 5 mins before the lab

**Pull request merged**